public/js/maps.js

```diff
@@ -2,10 +2,11 @@ const HOUSTON_COORDS = {lat: 29.7604, lng: -95.3698};
 
 let map;
 let bounds;
+let markers = [];
 let openMarkerWindow;
 
-/** 
- * Initialize the Google Map to be centerd in Houston and to add 
+/* 
```

Comment:
Nit: you actually want to keep the second ```*``` because it indicates that the comment is a JSDoc